include/swift/AST/DiagnosticsParse.def

```diff
@@ -1356,6 +1358,11 @@ ERROR(expected_type_after_as,none,
 ERROR(string_interpolation_extra,none,
       "extra tokens after interpolated string expression", ())
 
+// String interpolation isnt closed by a )
+// ie: `let west = "ye \("`
+ERROR(string_interpolation_unclosed, none,
+      "expected ')' at end of string interpolation", ())
+
```

Comment:
And this one.

Reply:
This one is used, 2 times in fact

Reply:
Oops, I mistook it for the one that provided the `')'` fix-it.

Reply:
😅